### PR TITLE
(CDAP-6385) Moved the FileSecureStore password from cdap-default.xml to

### PR DESCRIPTION
**Pull request by Nishith**
cdap-security.xml.

### CODE REVIEW

File: cdap-common/src/main/resources/cdap-default.xml [hunk 1]
```diff
@@ -1710,14 +1710,6 @@
   </property>
 
   <property>
```
Comment (bdmogal) [inline cdap-common/src/main/resources/cdap-default.xml:1712]: you will also have to update https://github.com/caskdata/cdap/blob/release/3.5/cdap-docs/admin-manual/build.sh#L22 with the MD5 of the file after this change

File: cdap-security/src/test/resources/cdap-security.xml [hunk 2]
```diff
@@ -31,4 +31,9 @@
       <value>JKS</value>
       <description>Specifies the ssl keystore type</description>
     </property>
+    <property>
```
Comment (bdmogal) [inline cdap-security/src/test/resources/cdap-security.xml:34]: how does this file get included?

Comment (Nishith) [inline cdap-security/src/test/resources/cdap-security.xml:34]: It doesn't. This is only for testing. In deployment the user will have to create a cdap-security.xml and add the required properties. We don't want to provide a default password.

File: cdap-common/src/main/resources/cdap-default.xml (hunk 1 above)
Comment (Nishith) [inline cdap-common/src/main/resources/cdap-default.xml:1712]: Does that need to happen after this PR is merged? Or in this PR?

Comment (bdmogal) [inline cdap-common/src/main/resources/cdap-default.xml:1712]: in this PR, to avoid a docs build break after this PR is merged.

**Comment by bdmogal** [comment]
LGTM apart from the md5 change from me. But ask @wolf31o2 to take a quick look just to make sure everything is ok.

**Comment by bdmogal** [comment]
Also add jira to the commit message.

**Comment by Nishith** [comment]
Will do, in the squash message.

**Comment by wolf31o2** [comment]
👍 LGTM

**Pull request merged**